.buildbot/Jenkinsfile

```diff
@@ -90,7 +90,7 @@ def call() {
       MODELZOO_REPORTDIR    = 'modelzoo_report'
       MODELZOO_HTML         = 'modelzoo.html'
       MODELZOO_STDOUT       = 'modelzoo.stdout'
-      MODELZOO_PUBLISH      = "${JENKINS_HOME}/jobs/${JOB_NAME}/htmlreports/Model_20Zoo_20Report"
+      MODELZOO_PUBLISHDIR   = "${JENKINS_HOME}/jobs/${JOB_NAME}/htmlreports/Model_20Zoo_20Report"
```

Comment:
As discussed, this and some other changed files seem to be from a faulty merge. I checked for expected differences using:
```
git difftool -d `git merge-base origin/main HEAD` HEAD --
```
and:
```
(for i in  $(git diff --name-only `git merge-base origin/main origin/feature/onnx_to_torch` origin/feature/onnx_to_torch); do git diff `git merge-base origin/main origin/feature/onnx_to_torch` origin/feature/onnx_to_torch -- $i >old; git diff `git merge-base origin/main HEAD` HEAD -- $i >new; echo "### $i"; xxdiff -D old new; done)
```